scripts/config/templates/nginx.conf.erb

```diff
@@ -15,7 +15,7 @@ http {
   gzip_vary on;
   gzip_proxied any;
 
-  log_format custom_agent  'network.client.ip=$remote_addr timestamp=$time_iso8601 log_source="nginx" '
+  log_format custom_agent  'network.client.ip=$remote_addr ip.forward_chain=$http_x_forwarded_for timestamp=$time_iso8601 log_source="nginx" '
```

Comment:
We'll want to probably nest the attribute (for datadog) in `network.client.ip_forward_chain` or something like that